server/utils/cloudinary.js

```diff
@@ -2,9 +2,9 @@ require("dotenv").config();
 
 const cloudinary = require("cloudinary").v2;
 cloudinary.config({
-  cloud_name: process.env.CLOUDINARY_NAME,
-  cloud_name: process.env.CLOUDINARY_API_KEY,
-  cloud_name: process.env.CLOUDINARY_API_SECRET,
+    cloud_name: process.env.CLOUDINARY_NAME,
+    api_key: process.env.CLOUDINARY_API_KEY,
+    api_secret: process.env.CLOUDINARY_API_SECRET,
```

Comment:
Let's update the sample.env with these new environment variables please. Just the key names not the values